extensions/cornerstone/jest.config.js

```diff
@@ -3,7 +3,6 @@ const pkg = require('./package');
 
 module.exports = {
   ...base,
-  name: pkg.name,
```

Comment:
name isn't a valid field, was causing spurious warnings during the unit test phase of this change.